src/interactions/panZoomInteraction.ts

```diff
@@ -439,6 +577,112 @@ namespace Plottable.Interactions {
     }
 
     /**
+     * Gets the minimum domain value for the scale, constraining the pan/zoom
+     * interaction to a minimum value in the domain.
+     *
+     * Note that this differs from minDomainExtent/maxDomainExtent, in that
+     * those methods provide constraints such as showing at least 2 but no more
+     * than 5 values at a time.
+     *
+     * By contrast, minDomainValue/maxDomainValue set a boundary beyond which
+     * the user cannot pan/zoom.
+     *
+     * @param {QuantitativeScale<any>} quantitativeScale The scale to query
+     * @returns {D} The minimum domain value for the scale.
+     */
+    public minDomainValue<D>(quantitativeScale: QuantitativeScale<D>): D;
```

Comment:
As with minDomainExtent/maxDomainExtent, you don't actually have to add the scale to the interaction before this. You could also add it after.